### PR TITLE
Update using-the-windows-namespace-with-unity-apps-for-hololens.md

### PR DESCRIPTION
**Pull request by dfields-msft**
Adding a section to this page to describe usage of the new Microsoft.Windows.MixedReality.DotNetWinRT package which enables use of Windows APIs in Unity scripts without preprocessor conditionals, including in Play Mode.

### CODE REVIEW

**Comment by dfields-msft** [comment]
@thetuvix please review these changes.

File: mixed-reality-docs/using-the-windows-namespace-with-unity-apps-for-hololens.md [hunk 1]
```diff
@@ -45,6 +45,14 @@ public class WinRTAPI : MonoBehaviour {
 
 When you double-click a script in the Unity editor, it will by default launch your script in an editor project. The WinRT APIs will appear to be unknown because the Visual Studio project does not reference the Windows Runtime. Further, the **ENALBE_WINMD_SUPPORT** directive will be undefined and any *#if* wrapped code will be ignored until you build your project into a UWP Visual Studio solution.
 
+## Alternative for Mixed Reality APIs
```
Comment (thetuvix) [inline mixed-reality-docs/using-the-windows-namespace-with-unity-apps-for-hololens.md]: Should we put this section at the top of the page, suggesting this approach as the primary recommended path for consuming WinRT APIs in Unity?  We can mention the key caveat for now that the API projection is incomplete - however, for any APIs present in the projection, this is the way we suggest everyone to go.

Comment (dfields-msft) [reply mixed-reality-docs/using-the-windows-namespace-with-unity-apps-for-hololens.md]: I debated about that, and I'm willing to go either way - I elected to keep it secondary for now for two main reasons: 1) it's brand new, and may have unforeseen limitations or bugs and 2) the gaps include significant platform APIs like Windows.Storage.StorageFile which we will never incorporate (until a comprehensive XLang package supercedes this one). However, if you are feeling more bullish about it, I'm willing to move it to the top. We could also do this later once it's got a bit more mileage.

Comment (dfields-msft) [reply mixed-reality-docs/using-the-windows-namespace-with-unity-apps-for-hololens.md]: On further reflection, the way I wrote it I think it is pretty clearly scoped to MR, so #2 is less of a concern. I'll try promoting it to the top...

**Comment by dfields-msft** [comment]
@jessemcculloch I think this is ready to merge; @thetuvix has signed off. Something seems wedged with the "OpenPublishing.Build (2 of 3) Preparing" check, but everything else has passed...

**Pull request merged**